.gitignore

```diff
@@ -0,0 +1,59 @@
+# Logs
```

Comment:
you should be just setting this globally and not on each project

Reply:
This is just github's stock .gitignore for node.

Reply:
yeah which you should have as your global .gitignore

Reply:
I don't understand, are you suggesting this project *shouldn't* have a .gitignore?

Reply:
yes, I only put a .gitignore into a project is if their is a project specific set of things that need to be ignored and I have a global .gitignore for language level stuff (and you should too).

Reply:
Language-level stuff is project specific. My global ignore contains OS & editor stuff.

Reply:
Honestly, if you don't like my PR, just reject it and change https://github.com/calvinmetcalf/rollup-plugin-node-globals/pull/11/files#diff-d5a8dcb6650763d61c3ffe20971461fbL75 yourself so that your plugin doesn't fall over as soon as it sees `async`.